Changelog.md

```diff
@@ -1,5 +1,53 @@
 # Changelog
 
+### 1.0.0
+**This is a breaking change**
+
+This release only supports Kubernetes versions >= v1.19. The support for Ingress Object in `networking.k8s.io/v1beta` is being dropped and manifests should now use `networking.k8s.io/v1`.
+
+**Image:**
+
+- `k8s.gcr.io/ingress-nginx/controller:v1.0.0@sha256:0851b34f69f69352bf168e6ccf30e1e20714a264ab1ecd1933e4d8c0fc3215c6`
+
+**Known Issues**
+Ingress Controller only supports cluster scoped IngressClass and needs cluster wide permission for this object, otherwise it is not going to start.
```

Comment:
we have an open issues for this, can we add it here.  https://github.com/kubernetes/ingress-nginx/issues/7502

Reply:
It's actually #7510 
But my personal opinion, we CAN merge this and add the issue as a follow up, wdyt? (otherwise all the CI is going to run again...)

Reply:
#7510 as well

Reply:
@nehaLohia27 please open a follow UP PR later adding the description proposed by @strongjz

Reply:
sure @rikatz would open a follow up PR.

Reply:
opened follow up PR #7536